.changeset/flat-snails-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-clickable": patch
+---
+
+Remove uses of enzyme from wonder-blocks-clickable tests
```

Comment:
**suggestion:** These changes don't require to release a new version, mostly because we are not making any changes on the code released to users (only testing changes).

I think that for that case, you can just use `yarn changeset --empty` so the GH action won't complain.

Reply:
Good call.  I'll do that for all of the test-only changes.